packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -95,6 +133,36 @@ export const usePeoplePane = (props: {
     setShowMuteAllPrompt(false);
   }, [onMuteAllRemoteParticipants, setShowMuteAllPrompt]);
 
+  /* @conditional-compile-remove(media-access) */
+  const [showForbidOthersAudioPrompt, setShowForbidOthersAudioPrompt] = React.useState(false);
```

Comment:
nit: Lets add an import for `useState` and remove the React keyword

Reply:
Done.